docs/vc-http-api.yml

```diff
@@ -89,6 +89,30 @@ paths:
           description: invalid input!
         "500":
           description: error!
+  /credentials/verifyMany:
+    post:
+      tags:
+        - Verifier
+      summary: Verifies a list of verifiableCredentials and returns a list of index matched verificationResults in the response body.
+      operationId: verifyCredentials
+      description: Verifies a list of verifiableCredentials and returns a list of index matched verificationResults in the response body.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/VerifyCredentialsRequest"
+        description: Parameters for verifying a list of verifiableCredentials.
+      responses:
+        "200":
+          description: Verifiable Credentials successfully verified!
```

Comment:
```suggestion
          description: The array of Verifiable Credentials was processed. NOTE: Some verifications might have failed; successful processing does not mean successful verification.
```

Reply:
No need to use exclamation marks in API descriptions. None of this should be a surprise or surprising. :)

Reply:
totally understand, just replicated what was in the existing api for `credentials/verify`.  have updated this.  Also really like adding the additional clarification to the 200 response and have incorporated this